skins/cat17/src/app/lib/validation/bank_data_validator.js

```diff
@@ -26,21 +26,21 @@ export class BankDataValidator {
 	}
 
 	/**
-	 * @param {string} accountNumber
-	 * @param {string} bankCode
+	 * @param {string} accountId
+	 * @param {string} bankId
 	 * @return {Promise}
 	 */
-	validateClassicBankData( accountNumber, bankCode ) {
-		accountNumber = accountNumber.replace( /\s/g, '' );
-		bankCode = bankCode.replace( /\s/g, '' );
-		if ( accountNumber === '' || bankCode === '' ) {
+	validateClassicBankData( accountId, bankId ) {
```

Comment:
I'm not sure if we should rename the fields - yes, it's more consistent with `validateBankData`, but at the same time, the "domain language" of the classic bank data - at least in the translation from German - would be ""account number" and "bank code".